common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -163,6 +163,9 @@ export interface ISequencedDocumentMessage {
 
     // Timestamp when the server ticketed the message
     timestamp: number;
+
+    // Experimental field for storing the rolling hash at sequence number
+    expHash1: string;
```

Comment:
Should this be optional? otherwise, it won't be backward compatible.

Reply:
Its compatible cause no one is using it. But optional is safer